src/ir_Teco.cpp

```diff
@@ -304,19 +293,19 @@ stdAc::fanspeed_t IRTecoAc::toCommonFanSpeed(const uint8_t speed) {
 
 /// Convert the current internal state into its stdAc::state_t equivalent.
 /// @return The stdAc equivalent of the native settings.
-stdAc::state_t IRTecoAc::toCommon(void) {
+stdAc::state_t IRTecoAc::toCommon(void) const {
   stdAc::state_t result;
   result.protocol = decode_type_t::TECO;
   result.model = -1;  // Not supported.
-  result.power = this->getPower();
-  result.mode = this->toCommonMode(this->getMode());
+  result.power = _.Power;
+  result.mode = toCommonMode(_.Mode);
   result.celsius = true;
-  result.degrees = this->getTemp();
-  result.fanspeed = this->toCommonFanSpeed(this->getFan());
-  result.swingv = this->getSwing() ? stdAc::swingv_t::kAuto :
+  result.degrees = getTemp();
+  result.fanspeed = toCommonFanSpeed(_.Fan);
+  result.swingv = _.Swing ? stdAc::swingv_t::kAuto :
                                      stdAc::swingv_t::kOff;
```

Comment:
This might fit on a single line now. If not, please align the `stdAc`s.
eg.
```
result.swingv = _.Swing ? stdAc::swingv_t::kAuto : stdAc::swingv_t::kOff;
```
or
```
result.swingv = _.Swing ? stdAc::swingv_t::kAuto
                        : stdAc::swingv_t::kOff;
```